simpl/tests/test_external_schema.py

```diff
@@ -345,3 +469,44 @@ def test_run_resolve_instances(self):
 
         # There should be 5 instances
         self.assertEqual(len(instances), 5)
+
+    def test_number_of_queries_with_multiple_instances_and_players(self):
+        run = baker.make(Run, game=self.game)
+
+        number_instances = 120
+        number_players_per_instance = 5
+
+        instances = []
+        for _ in range(number_instances):
+            instances.append(baker.make(Instance, run=run))
+
+        # Create n users with Auth0 IDs
+        users = []
+        for i in range(number_players_per_instance):
+            users.append(baker.make(SocialAccount, provider="auth0", uid=f"auth0_{i}"))
+
+        # Create characters linking the users to the instance
+        for instance in instances:
+            for user in users:
+                baker.make("simpl.Character", instance=instance, user=user.user)
+
+        # All of this should take only 2 queries
+        with self.assertNumQueries(5):
```

Comment:
how many? :smile:
(tbh, I'm not sure we necessarily need these comments at all. I reckon `with self.assertNumQueries(5):` is pretty self-explanatory)

Reply:
I should make a script to grep these debugging statements, I always forget to remove them :see_no_evil: thanks for catching that